contracts/DCAHub/DCAHubPositionHandler.sol

```diff
@@ -196,6 +197,8 @@ abstract contract DCAHubPositionHandler is ReentrancyGuard, DCAHubConfigHandler,
       _increase ? IDCAPermissionManager.Permission.INCREASE : IDCAPermissionManager.Permission.REDUCE
     );
 
+    if (_increase && (!_allowedTokens[_userPosition.from] || !_allowedTokens[_userPosition.to])) revert IDCAHubConfigHandler.UnallowedToken();
```

Comment:
I think this should be in `increasePosition` instead of here

Reply:
Nit: maybe we could create an internal function that checks 
```if (!_allowedTokens[_from] || !_allowedTokens[_to]) revert IDCAHubConfigHandler.UnallowedToken();```

Reply:
@nchamo the thing is that if I check this in `increasePosition` I will have to access to user position's through storage => costly.

Reply:
Oh, I see. Ok, makes sense then